app/lib/widgets/wallets/add_wallet.dart

```diff
@@ -26,6 +31,40 @@ class _NewWalletState extends State<NewWallet> {
   bool saveLoading = false;
   String? nameError;
   String? secretError;
+
+  String? generateHexSeed(String secret) {
+    String? hexSeed = '';
+
+    if (' '.allMatches(secret).length == 23) {
+      final entropy = bip39.mnemonicToEntropy(secret);
+      final seedList = hex.decode(entropy).toList();
+      seedList.addAll([0, 0, 0, 0, 0, 0, 0, 0]);
+      hexSeed = '0x${Blake2b(32).hex(seedList)}';
+    } else if (' '.allMatches(secret).length == 11) {
+      final entropy = bip39.mnemonicToEntropy(secret);
+      hexSeed = '0x${entropy.padRight(64, '0')}';
```

Comment:
in case we used 12 words, then we used the hex seed for this mnemonic, it will add it.